### DISCUSSION TITLE
Haade.fr fait peau neuve / gets a makeover

### DISCUSSION THREAD

**Discussion by haade-administrator** [haade-administrator/haade.fr#2] [Annonces / Announcements]
🆕 **Après presque 1 an d’absence, Haade.fr fait peau neuve.** 
Après de dures labeurs j'ai décider de reprendre le site en profondeur, en effet fini wordpress et ses contraintes d'hébergement, voilà un site ssg plus rapide et entièrement dépendant des services github.
Hébergé sur leurs serveurs, j'utilise le fil commentaires et discussions afin de bénéficier de leurs atouts et de la gratuité. Tout n'est pas parfait et terminé mais ça va suivre son cours, en attendant je vais reprendre les tutos en essayant d'être le plus pertinent possible ce qui est loin d'être évident.

======

🆕 **After almost a year of absence, Haade.fr is getting a makeover.**
After hard work I decided to take over the site in depth, indeed finished wordpress and its hosting constraints, here is a faster ssg site and entirely dependent on github services.
Hosted on their servers, I use the comments and discussions thread in order to benefit from their advantages and free access. Everything is not perfect and finished but it will take its course, in the meantime I will resume the tutorials trying to be as relevant as possible which is far from obvious.

**Comment by titis14** [comment]
Cool tout ça je me lance dans HA et je viens de commander ce qu'il faut pour le tuto pour connecter le linky.

**Reply by haade-administrator** [reply]
bientôt un article sur le suivi de consommations de l'eau avec un circuit imprimé ?
